stdlib/3/struct.pyi

```diff
@@ -1,30 +1,35 @@
 # Stubs for struct
 
 # Based on http://docs.python.org/3.2/library/struct.html
+# Based on http://docs.python.org/2/library/struct.html
 
-from typing import overload, Any, Tuple
+import sys
+from typing import Any, Tuple, Text, Union
 
 class error(Exception): ...
 
-def pack(fmt: str, *v: Any) -> bytes: ...
-# TODO buffer type
-def pack_into(fmt: str, buffer: Any, offset: int, *v: Any) -> None: ...
+_FmtType = Union[bytes, Text]
+if sys.version_info >= (3,):
+    _BufferType = Union[bytes, bytearray, memoryview]
```

Comment:
Yes, I'd prefer `array[Any]` for explicitness; at some point I'd like to make sure typeshed doesn't use any generic types without type parameters.